lib/core/elastic_connector_actions.rb

```diff
@@ -189,13 +189,15 @@ def create_job(connector_settings:)
           status: Connectors::SyncStatus::PENDING,
           created_at: Time.now,
           last_seen: Time.now,
+          trigger_method: connector_settings.sync_now? ? 'on-demand' : 'scheduled',
```

Comment:
'on-demand' and 'scheduled' should be constants

Reply:
Added a new enum